artifactory_cleanup/rules/docker.py

```diff
@@ -45,9 +45,8 @@ def _manifest_to_docker_images(self, artifacts: ArtifactsList):
             # already done it or it's just a folder
             if "name" not in artifact or artifact["name"] != self.MANIFEST_FILENAME:
                 continue
-
-            artifact["path"], docker_tag = artifact["path"].rsplit("/", 1)
-            artifact["name"] = docker_tag
+            artifact["path"] = artifact["properties"]["docker.repoName"]
```

Comment:
I'm not sure how it's saved nowadays, and that all properties exists on all docker images - could you add a false back instead?
The logic would be this one - if `rsplit` returns two elements - we use old logic, if not - new one.
And please add two tests for that as well, so we know that it works as it used to with old artifacts too